abstract.md

```diff
@@ -0,0 +1,41 @@
+# Making research software more robust
+Morgan Taschuk and Greg Wilson
+
+
+At the cutting edge of research, bioinformatics software is often written by a
+single person who is both developer and scientist, producing insightful, novel
+results for their own projects. Upon presenting their work, other
+people will want to use their methods. But much of academic software is written
+in a prototypic, exploratory and rigid style that is incompatible with reuse. Often, the end user will
+need to spend a considerable amount of time either reading code or in
+communication with the original developers. More times than we like to admit,
+we will simply give up on the software, either writing our own or finding
```

Comment:
"we will simply give up and either write our own or find something else that accomplishes..."